src/systems/filecoin_mining/sector/sealing.id

```diff
@@ -33,14 +31,25 @@ type OnChainSealVerifyInfo struct {
 // for a sector, for example PoSt.
 // These should be stored and indexed somewhere by CommR.
 type ProofAux struct {
-    CommRLast             SealedSectorCID
+    CommRLast             Commitment
     CommC                 Commitment
 
     // TODO: This may be a partially-cached tree.
     // this may be empty
     CachedMerkleTreePath  file.Path
 }
 
+type ProofAuxTmp struct {
```

Comment:
since we now have two of these, what about naming them for the use:
```
type SealVerifyAux struct{ ... } // aux info to verify SealOutputs with
type SealProofVerifyAux struct { ... } // aux info to verify a CreateSealProofOutputs with
```

Reply:
Mmmm, that's not quite what they are though. They're both produced by `SealSector`. One holds things which will be required for `CreateSealProof`. The other holds the subset of those things which will be needed later when generating `PoSt`.